#! /bin/sh -e
## 3_newpatch.dpatch by Roland Mas <lolando@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: readme-debian.el: remove automode from file (moved to startup script instead)

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
diff -urNad 3.tmp/elisp/readme-debian.el 3/elisp/readme-debian.el
--- 3.tmp/elisp/readme-debian.el	2003-01-05 17:10:41.000000000 +0100
+++ 3/elisp/readme-debian.el	2003-01-05 17:29:40.000000000 +0100
@@ -34,10 +34,6 @@
  Value of `user-mail-address' variable,
  Value returned by the `user-mail-address' function.")
 
-(add-to-list 'auto-mode-alist '("debian/README.Debian$" . readme-debian-mode))
-(add-to-list 'auto-mode-alist '("^/usr/share/doc/.*/README.Debian.*$" . readme-debian-mode))
-
-
 (defun readme-debian-update-timestamp ()
   "Function to update timestamp in README.Debian files, automatically invoked when saving file"
   (save-excursion
